datahub-web-react/src/app/entity/EntityRegistry.tsx

```diff
@@ -128,6 +128,10 @@ export default class EntityRegistry {
                         entity: relationship.entity as EntityInterface,
                         type: (relationship.entity as EntityInterface).type,
                     })),
+                downstreamRelationships: genericEntityProperties?.downstream?.relationships
```

Comment:
minor: we should probably handle this filtering at the backend layer if we do not

Reply:
yeah I'm pretty sure we do, but I'm just mimicking the existing `downstreamChildren` above to be safe